.gitignore

```diff
@@ -1,5 +1,15 @@
 .DS_Store
+.idea
 bower_components/
 node_modules/
+test/*.js*
+#util.js is written in .js for gulp to use, so we can't typescript that yet
+lib/ast.js
+lib/bolt.js
+lib/file-io.js
+lib/firebase-rest.js
+lib/rules-generator.js
+lib/rules-parser.js 
+lib/simulator.js
 dist/
```

Comment:
Maybe we should maintain the strict pipeline.  Files in src are original src files, lib are all js file (maybe copied from src, may generated via PEG or TS).  That way /lib just gets git-ignored completely - but we use as publish point for node-compatible modules (or we send to /dist instead to be more like other repos).